resources/lang/zh-TW.json

```diff
@@ -1,4 +1,5 @@
-{
+{   
```

Comment:
Hi @jasoncheng7115 
You have trailing spaces to remove here

Reply:
Thank you, I have removed the extra space